### PR TITLE
Add UK south

### PR DESCRIPTION
**Pull request by Kalmis**
The UK south cluster is missing FHIR-converter
TODO:

- [ ] Create `fhir-converter` namespace

### CODE REVIEW

File: charts/fhir-converter/values-azure-prod-uk.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fhir:
+  image:
+    repository: globalkaikuacr.azurecr.io/fhir-converter
+  certIssuer:
+    enabled: true
+
+ingress:
+  cloud: "azure"
+  host: "fhir-converter-us.kaikuhealth.com"
```
Comment (hkotka) [inline charts/fhir-converter/values-azure-prod-uk.yaml:9]: Ingress hostname should be perhaps `fhir-converter-uk.kaikuhealth.com`?

**Review by hkotka — CHANGES_REQUESTED** [review]
Looks otherwise good, but ingress should have unique name.

File: charts/fhir-converter/values-azure-prod-uk.yaml (hunk 1 above)
Comment (Kalmis) [reply charts/fhir-converter/values-azure-prod-uk.yaml]: Yes indeed, fixed!

**Review by hkotka — APPROVED** [review]
LGTM

**Pull request merged**